examples/hybridexampleapp/android/app/build.gradle

```diff
@@ -0,0 +1,76 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+    id 'com.facebook.react'
+
+}
+
+android {
+    signingConfigs {
+        release {
+            storeFile file('/Users/yrn/Desktop/release')
+            storePassword 'qwertttt'
+            keyAlias 'key0'
+            keyPassword 'qwertttt'
+        }
```

Comment:
I guess we can either remove the `/Users/urn/Desktop/release` keystore file reference and add a local debug keystore which we can use for release and debug (like in `examples/default`), what do you think?